hyperspy/gui/__init__.py

```diff
@@ -38,7 +38,7 @@ def set_ets_toolkit(toolkit):
 _logger.debug('Current MPL backend: %s', backend)
 if "WX" in backend:
     set_ets_toolkit("wx")
-elif "Qt" in backend:
+elif "Qt4" in backend:
```

Comment:
Is it fixing #1559? If so, does it mean that instead of using `%matplotlib qt`, we should use `%matplotlib qt4`, before importing hyperspy?